backend/src/main/java/clf/integra/backend/controller/DepositController.java

```diff
@@ -1,22 +1,37 @@
 package clf.integra.backend.controller;
 
 import clf.integra.backend.dto.DepositImportRequest;
+import clf.integra.backend.dto.DepositsDTO;
 import clf.integra.backend.service.DepositsService;
 import jakarta.validation.Valid;
 import lombok.AllArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.security.access.prepost.PreAuthorize;
 
+import java.util.UUID;
+
 @RestController
 @AllArgsConstructor
 @CrossOrigin("*")
 public class DepositController {
     private DepositsService depositsService;
 
+    @PostMapping("/deposits/create/{userId}")
```

Comment:
you don't need to specify /create, as in theory the action is define by the verb of the request `POST`